lib/dotcom_web/components/system_status/status_label.ex

```diff
@@ -22,11 +22,14 @@ defmodule DotcomWeb.Components.SystemStatus.StatusLabel do
     ~H"""
     <span class={[status_classes(@status), "flex items-center gap-2"]}>
       <.status_icon status={@status} />
-      {@rendered_prefix} {description(@status, @plural)}
+      {@rendered_prefix} {expectation(@status, @prefix)} {description(@status, @plural)}
     </span>
     """
   end
 
+  defp expectation(:delay, prefix) when is_binary(prefix), do: "Expect"
```

Comment:
Non-blocking: This is a weird and wacky enough use case that it probably deserves both a test and an explanatory comment (or at least one or the other of those 😅).